src/gc.c

```diff
@@ -22,6 +22,18 @@
 #include "datastructure.h"
 // logg_rate_limit_message()
 #include "database/message-table.h"
+// get_nprocs()
+#include <sys/sysinfo.h>
+// get_filepath_usage()
+#include "files.h"
+
+// Warn if space in a storage place exceeds this threshold
+// default: 90%
+#define WARN_LIMIT 90
```

Comment:
Is this still needed?

The threshold for `DISK` and `SHMEM` is set now in `config.c` (or read from `pihole-FTL.conf`)